composer.json

```diff
@@ -32,39 +32,39 @@
         "doctrine/doctrine-fixtures-bundle": "^2.3",
         "doctrine/doctrine-migrations-bundle": "^1.2",
         "doctrine/orm": "^2.5",
-        "friendsofsymfony/oauth-server-bundle": "^1.5",
+        "friendsofsymfony/oauth-server-bundle": "^1.6@dev",
         "friendsofsymfony/rest-bundle": "^2.1",
         "fzaninotto/faker": "^1.6",
         "gedmo/doctrine-extensions": "^2.4.12",
-        "incenteev/composer-parameter-handler": "^2.1",
+        "incenteev/composer-parameter-handler": "^2.1.3",
         "jms/serializer-bundle": "^2.0",
         "knplabs/knp-gaufrette-bundle": "^0.5",
         "knplabs/knp-menu-bundle": "^2.1",
-        "liip/imagine-bundle": "^1.9.1",
+        "liip/imagine-bundle": "^2.0@dev",
         "ocramius/proxy-manager": "^2.1",
         "payum/payum": "^1.4",
         "payum/payum-bundle": "^2.2",
         "php-http/guzzle6-adapter": "^1.1",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
         "ramsey/uuid": "^3.7",
-        "sensio/distribution-bundle": "^5.0.21",
         "sonata-project/block-bundle": "^3.3",
         "sonata-project/intl-bundle": "^2.2",
         "stof/doctrine-extensions-bundle": "^1.2",
         "swiftmailer/swiftmailer": "^6.0",
-        "sylius-labs/association-hydrator": "^1.0",
+        "sylius-labs/association-hydrator": "^1.1",
+        "sylius-labs/sensio-distribution-bundle": "^6.0",
```

Comment:
I've forked DistributionBundle because issue about Symfony 4 support has been immediately closed, so we have to either use Flex (whereas we need only script executor from there now) or our DistributionBundle fork (but we take the maintenance burden, while we're lacking human resources for that).

Reply:
See sensiolabs/SensioDistributionBundle#334.

Reply:
@pamil Where is the fork? :)